scripts/kube-init.sh

```diff
@@ -32,7 +32,8 @@ setenforce 0
 
 # Mount root to fix dns issues
 # Define $HOME since somehow this is not defined
-HOME=/home/travis
+# Changed from travis to GH Actions agent default user
+#HOME=/home/runner 
```

Comment:
Is this line intentionally commented out?

Reply:
https://github.com/kubernetes-client/python/pull/1659#pullrequestreview-840025736